product_configurator/models/product_attribute.py

```diff
@@ -183,16 +181,18 @@ def onchange_values(self):
     def _check_default_values(self):
         """default value should not be outside of the
         values selected in attribute line"""
-        for line in self.filtered(lambda l: l.default_val):
+        for line in self.filtered(lambda line: line.default_val):
```

Comment:
I'm pretty sure these changes were already in v16.
Can you check you brought over all commits there?